OJP/OJP_Status.xsd

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xs:schema xmlns="http://www.vdv.de/ojp" xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:siri="http://www.siri.org.uk/siri" targetNamespace="http://www.vdv.de/ojp" elementFormDefault="qualified" attributeFormDefault="unqualified" id="OJP_siri_service">
+	<xs:annotation>
+		<xs:documentation>OJP/OJP_Status.xsd - OJP Status request and response</xs:documentation>
+	</xs:annotation>
+	<xs:import namespace="http://www.siri.org.uk/siri" schemaLocation="../siri/siri_model/siri_all-v2.0.xsd"/>
+	<xs:include schemaLocation="OJP_All.xsd"/>
+	<xs:import namespace="http://www.siri.org.uk/siri" schemaLocation="../siri/siri_model/siri_modes-v1.1.xsd"/>
+	<xs:annotation>
+		<xs:documentation>=============================== Status Service ==========================================</xs:documentation>
+	</xs:annotation>
+	<xs:element name="OJPStatusResponse" type="OJPStatusResponseStructure" substitutionGroup="siri:AbstractFunctionalServiceCapabilitiesResponse"/>
+	<xs:complexType name="OJPStatusResponseStructure">
+		<xs:complexContent>
+			<xs:extension base="siri:AbstractServiceCapabilitiesResponseStructure">
+				<xs:sequence>
+					<xs:group ref="ServiceResponseContextGroup"/>
```

Comment:
Is a problem structure needed here?

Reply:
A somehow similar question: What happens if the service hasn't started? If we didn't get any response in that case, then it wouldn't be possible or necessary to signal that in the problem structure. 

Second question: What will be the value of *ServiceReady* if it is not ready?

Reply:
Oops - I just see that my second question was a silly question: the element *ServiceReady* would just be missing in the case mentioned.

Reply:
@sgrossberndt  I could add it (because all the other have one), but what would I put in it? What errors could occur?

@trurlurl : The service started is more an information that is needed to check if the client should be restarted as well. E.g. because it cached some values.  And if the service really was not started, then there will be an http error anyhow.

Reply:
@ue71603 The only use for an error message I come about: *OK* and *SERVICE_DEPRECATED* to announce a phasing-out of the service.

Reply:
added problem structure